.rubocop.yml

```diff
@@ -26,10 +29,10 @@ Metrics/PerceivedComplexity:
 Metrics/CyclomaticComplexity:
   Enabled: false
 
-Layout/ParameterAlignment:
+Style/Documentation:
   Enabled: false
 
-Style/Documentation:
+Style/DoubleNegation:
```

Comment:
We find this dangerous inside an algorithm because `== true` is a lot more expressive.